docs/next/guides/columns/column-width.md

```diff
@@ -20,7 +20,7 @@ tags:
 
 ## Overview
 
-By default, the column width adjusts to the width of the content. However, if the width of the column content is less than `50px`, including `1px` for borders on the sides, the column width remains constant at `50px`. The column size can be passed as a `constant`, an `array` or a `function`.
+By default, the column width adjusts to the width of the content. However, if the width of the column content is less than `50px`, including `1px` for borders on the sides, the column width remains constant at `50px`. The column size can be passed as a constant, an array, or a function.
 
 The content inside a cell will be wrapped if it doesn't fit the cell's width.
```

Comment:
I don't see a contradiction here: the width of the column is one thing, and „the width of the content” (i.e. the space that the content takes horizontally, when there's no line break) is another.

There are other problems here (e.g., „content” mistaken for „contents”), but I believe rewriting this is out of scope for this PR.

Reply:
I would remove "column" from the sentence "However, if the width of the column content" because it's a cell's content, not a column's; and in the first sentence we don't use that as a fixed phrase.

Reply:
That's right, done: [6df3522](https://github.com/handsontable/handsontable/pull/9374/commits/6df3522d5b8e6b4e59b27f9de6a8464bace20e29)